common/oatbox/reporting/Report.php

```diff
@@ -47,5 +48,67 @@
  */
 class Report extends common_report_Report
 {
+    /** @var string */
+    private $interpolationMessage;
 
+    /** @var array */
+    private $interpolationData;
+
+    /**
+     * Create Report with translations support
+     *
+     * @throws common_exception_Error
+     */
+    public static function create(string $type, string $interpolationMessage, array $interpolationData = []): Report
```

Comment:
It was already released before: https://github.com/oat-sa/generis/commit/c52ae7a9e566f57d9a52bea39123f9d7ae412127#diff-0c5f5d447957a7012b605c9fea0336c1f61017340a68d0797fa0d789be48f74cR62